thorlcr/mfilemanager/thmfilemanager.cpp

```diff
@@ -374,6 +381,19 @@ class CFileManager : public CSimpleInterface, implements IThorFileManager
 
     IFileDescriptor *create(CJobBase &job, const char *logicalName, StringArray &groupNames, IArrayOf<IGroup> &groups, bool overwriteok, unsigned helperFlags=0, bool nonLocalIndex=false, unsigned restrictedWidth=0)
     {
+        if (isCloud())
```

Comment:
Would #ifdef _CONTAINERIZED be more sensible?

Reply:
I think I'll leave this as is for now.
It should really be a runtime isNas() I think, where normally true if _CONTAINERIZED, but in future if we support bare metal, me may want attached storage so same container with isNas()==false.